sheriff.go

```diff
@@ -84,6 +84,12 @@ func Marshal(options *Options, data interface{}) (interface{}, error) {
 		// we want the childs exposed at the toplevel to be
 		// consistent with the embedded json marshaller
 		if val.Kind() == reflect.Ptr {
+			if val.IsNil() { // For pointer fields with nil value, and omitempty not set, we set to nil (null) in JSON
+				if !jsonOpts.Contains("omitempty") {
+					dest[jsonTag] = nil
+				}
+				continue
```

Comment:
Why use `continue` and not just `else`?

Reply:
Because once the field is handled we move on to the next... however, I think Robbie highlighting the same fix in the upstream repo is a better solution, so I'll apply that instead. Checking for other changes in the upstream in case it makes sense to just get the latest (although I don't like the idea of breaking things) 😉